src/data/nat/factorial.lean

```diff
@@ -81,25 +90,26 @@ by { convert factorial_lt _, refl, exact one_pos }
 lemma factorial_eq_one : n! = 1 ↔ n ≤ 1 :=
 begin
   split; intro h,
-  { rw [← not_lt, ← one_lt_factorial, h], apply lt_irrefl },
-  { cases h with h h, refl, cases h, refl }
+  { rw [← not_lt, ← one_lt_factorial, h],
+    apply lt_irrefl },
+  cases h with h h, refl, cases h, refl,
 end
 
 lemma factorial_inj (h0 : 1 < n!) : n! = m! ↔ n = m :=
 begin
   split; intro h,
-  { rcases lt_trichotomy n m with hnm|hnm|hnm,
+  { obtain hnm | hnm | hnm := lt_trichotomy n m,
     { exfalso, rw [← factorial_lt, h] at hnm, exact lt_irrefl _ hnm,
       rw [one_lt_factorial] at h0, exact lt_trans one_pos h0 },
     { exact hnm },
-    { exfalso, rw [← factorial_lt, h] at hnm, exact lt_irrefl _ hnm,
-      rw [h, one_lt_factorial] at h0, exact lt_trans one_pos h0 }},
-  { rw h }
+    exfalso, rw [← factorial_lt, h] at hnm, exact lt_irrefl _ hnm,
+      rw [h, one_lt_factorial] at h0, exact lt_trans one_pos h0 },
+  rw h,
```

Comment:
These braces should be there.

Reply:
Yes. You can learn the mathlib style guidelines here: https://leanprover-community.github.io/contribute/style.html, which confirms this.